kibana-reports/server/routes/utils/visual_report/visualReportHelper.ts

```diff
@@ -131,19 +131,24 @@ export const createVisualReport = async (
   let buffer: Buffer;
   // remove top nav bar
   await page.evaluate(
-    /* istanbul ignore next */
-    () => {
-      // remove buttons
-      document
-        .querySelectorAll("[class^='euiButton']")
-        .forEach((e) => e.remove());
-      // remove top navBar
-      document
-        .querySelectorAll("[class^='euiHeader']")
-        .forEach((e) => e.remove());
-      document.body.style.paddingTop = '0px';
-    }
+      /* istanbul ignore next */
+      () => {
+        // remove buttons
+        document
+            .querySelectorAll("[class^='euiButton']")
+            .forEach((e) => e.remove());
+        // remove query bar
+        document
+            .querySelectorAll("[class='globalQueryBar']")
```

Comment:
I think this `globalQueryBar` also includes the time range. We still need to display that in the pdf/png.